musicbrainz/seeding.ts

```diff
@@ -145,6 +145,9 @@ export function convertLinkType(entityType: EntityType, linkType: LinkType, url?
 			if (url?.hostname.endsWith('.bandcamp.com')) {
 				return typeIds.bandcamp;
 			}
+			if (url?.hostname.replace('www.', '') == 'soundcloud.com') {
```

Comment:
Do the URLs ever have a www subdomain? The provider implementation doesn't accept it at least.